src/utilities/util_verify.c

```diff
@@ -122,14 +136,40 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
             goto err;
         }
     }
-    if ((ret = session->verify(session, uri, config)) != 0)
-        (void)util_err(session, ret, "session.verify: %s", uri);
-    else {
-        /*
-         * Verbose configures a progress counter, move to the next line.
-         */
-        if (verbose)
-            printf("\n");
+
+    /* Verify all the tables if no particular URI is specified. */
+    if (argc < 1) {
+        /* Open the metadata file and iterate through its entries, verifying each one. */
+        if ((ret = session->open_cursor(session, WT_METADATA_URI, NULL, NULL, &cursor)) != 0) {
+            /*
+             * If there is no metadata (yet), this will return ENOENT. Treat that the same as an
+             * empty metadata.
+             */
+            if (ret == ENOENT)
+                ret = 0;
+
+            ret = util_err(session, ret, "%s: WT_SESSION.open_cursor", WT_METADATA_URI);
```

Comment:
Can we use `WT_ERR` here?

Reply:
Yeah, I can give this a try, for some reason I thought that wasn't supported in utility files.